plugins/gradle-extensions-plugin/src/main/kotlin/com/github/vlsi/gradle/properties/dsl/ProjectExtensions.kt

```diff
@@ -78,11 +96,21 @@ class PropertyMapper internal constructor(private val project: Project) {
     fun string(name: String, default: String = "") =
         project.stringProperty(name, false) ?: default
 
-    fun int(name: String, default: Int = 0) =
-        project.stringProperty(name, false)?.toInt() ?: default
+    fun int(name: String, default: Int = 0) = project.stringProperty(name, false)?.let { value ->
+        value.toIntOrNull() ?: null.also {
+            project.logger.debug("Unable to parse $name=$value as Int, using default value: $default")
```

Comment:
What do you think of failing instead of silently converting the value to the default?